SpaceGame/Assets/Scripts/BuoyScripts/BuoySway.cs

```diff
@@ -20,6 +17,7 @@ public class BuoySway : MonoBehaviour
     
     private void Awake()
     {
+        m_collisionHandler = GetComponent<MotherShipCollisionHandler>();
```

Comment:
No, this is not a applicable fix, this makes the buoys rotate sideways instead of swaying